src/main/java/hw_2/Task2.java

```diff
@@ -0,0 +1,10 @@
+package hw_2;
+
+public class Task2 {
+    public static void main(String[] args) {
+        int x = 3;
+        int y = 4;
+        System.out.println(x + y + " " + x+y);
```

Comment:
should be extra space before and after '+'

Reply:
fixed